src/test/java/com/synopsys/integration/detect/FontLoaderTestIT.java

```diff
@@ -62,7 +65,9 @@ public void createTempDirectory() throws Exception {
         DetectConfigurationFactory detectConfigurationFactory = new DetectConfigurationFactory(propertyConfiguration, new SimplePathResolver(), gson);
         ConnectionFactory connectionFactory = new ConnectionFactory(detectConfigurationFactory.createConnectionDetails());
         ArtifactResolver artifactResolver = new ArtifactResolver(connectionFactory, gson);
-        DetectFontInstaller installer = new DetectFontInstaller(artifactResolver);
+        InstalledToolManager installedToolManager = new InstalledToolManager();
```

Comment:
Seeing these two classes carried everywhere together makes me thing the `InstalledToolManager` should handle the locating as well. Nothing needs to change now, but I think that would be an improvement.

Reply:
Looked into that.  The road block is that whoever does the locating needs to know the permanent directoy path upon creation, which we don't know until DirectoryManager is created.  InstallToolManager needs to be created earlier, so that it can be passed throughout the run while still being available in Application to write the json file.